test/conftest.py

```diff
@@ -78,7 +82,7 @@ def aws_id(request):
 @pytest.fixture(scope='session')
 def tag(request, framework_version, processor, py_version):
     provided_tag = request.config.getoption('--tag')
-    default_tag = '{}-{}-py{}'.format(framework_version, processor, py_version)
+    default_tag = '{}-{}-{}'.format(framework_version, processor, py_version)
```

Comment:
are we tagging the mxnet containers without the 'py'?

Reply:
no, but I moved the 'py' to l. 54 so that way the pytest output shows 'py3' instead of just '3' (I think the former is more understandable)